docs/topics/testing/tools.txt

```diff
@@ -1905,6 +1952,8 @@ If you are testing from an asynchronous function, you must also use the
 asynchronous test client. This is available as ``django.test.AsyncClient``,
 or as ``self.async_client`` on any test.
 
+.. class:: AsyncClient(enforce_csrf_checks=False, raise_request_exception=True, *, headers=None, **defaults)
+
 ``AsyncClient`` has the same methods and signatures as the synchronous (normal)
 test client, with two exceptions:
```

Comment:
It looks like `**defaults` isn't documented here *at all*, or for `AsyncRequestFactory`. It's not API-compaitble - it's added into the ASGI scope in `AsyncRequestFactory`:

https://github.com/django/django/blob/37c5b8c07be104fd5288cd87f101e48cb7a40298/django/test/client.py#L576

I think we should add a line here, and in `AsyncRequestFactory` explaining this, something like:

```
Arbitrary keyword arguments in `defaults` are added directly into the ASGI
scope.
```

I think this can added to the "list of exceptions" in the docs here for `AsyncClient`, but we'd want to make it clear that we're talking about `defaults` for the initializer, and `follow`/`extra` for the methods.